stefanradev93/BayesFlow
# Python package build; License; Changelog; tox tests

Simplify installation and add meta data:
* Set up python package build via pyproject.toml, setup.cfg and setup.py (for dev build) files --> install via `pip install -e .`
* Add MIT license (can be changed for an alternative one if preferred)
* Add bayesflow/version.py and CHANGELOG.rst for versioning

Tests:
* Add tox.ini to run tests

To finalize:
* What should be the minimum python version? Currently 3.8
* What version? Currently 0.0.0 beta 1.

What could be done next:
* Add bayesflow as package to pypi (and add auto deploy via GitHub Actions)
* Run tests e.g. on GitHub Actions
* Add quality checks, e.g. flake8, black
* Some tests are failing
